src/Abbrev.ts

```diff
@@ -0,0 +1,10 @@
+// Abbreviations for entering dates with chrono
```

Comment:
This is great. After I merge this PR, I'll rename Abbrev to DateAbreviations, to leave room for other abbreviations in future...